### ISSUE TITLE
During the training, the loss and probs seem bad.

### ISSUE THREAD

**Issue by ceezy767** [HobbitLong/CMC#26]
After 126 epochs for training, the loss still seems huge. And the probs for "L","ab" are only about 0.007.  We set learning rate, batch size to 6e-2 and 1024 (8 Tesla-V100).

Train: [126][930/1252]	BT 0.827 (0.953)	DT 0.001 (0.234)	loss 6.161 (6.071)	l_p 0.007 (0.007)	ab_p 0.006 (0.006)
torch.Size([1024, 16385, 1])
Train: [126][940/1252]	BT 0.630 (0.951)	DT 0.001 (0.232)	loss 5.945 (6.071)	l_p 0.007 (0.007)	ab_p 0.006 (0.006)
torch.Size([1024, 16385, 1])

I don't know what's wrong with our experiment setting. Could you share the curves of training loss and probs of 'L' and 'ab'?

**Comment by chengchen77** [comment]
@ceezy767 Did you find out what's the problem?
